ci/meta.yml

```
@@ -24,6 +24,136 @@ meta:
         outputs:
           - name: terraform-config
       params:
-        AWS_ACCESS_KEY_ID: ((dataworks-secrets.ci_aws_access_key_id))
-        AWS_SECRET_ACCESS_KEY: ((dataworks-secrets.ci_aws_secret_access_key))
+        AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+        AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
         AWS_REGION: eu-west-2
+    terraform-apply:
+      task: terraform-apply
+      config:
+        platform: linux
+        image_resource:
+          type: docker-image
+          source:
+            repository: ((terraform.repository))
+            version: ((terraform12.version))
+            tag: ((terraform12.version))
+        params:
+          AWS_ACCESS_KEY_ID: ((ci.aws_access_key_id))
+          AWS_SECRET_ACCESS_KEY: ((ci.aws_secret_access_key))
+          TF_INPUT: false
+          TF_VAR_assume_role: ci
+          TF_CLI_ARGS_apply: -lock-timeout=300s
+          TF_CLI_ARGS_plan:  -lock-timeout=300s
+        run:
+          dir: 'dataworks-analytical-service-infra/terraform/deploy/infra'
+          path: sh
+          args:
+            - -exc
+            - |
+              cp ../../../../terraform-config/terraform.tf .
+              cp ../../../../terraform-config/terraform.tfvars .
+              terraform init
+              terraform workspace show
+              terraform plan -var assume_role=ci -out terraform.plan
```

```suggestion
              terraform plan -out terraform.plan
```
and use the `sed` command to remove assume role from bootstrap (same as the other pipelines)

---

Done 👍